typescript/segmented-cfmm/README.md

```diff
@@ -0,0 +1,74 @@
+# Segmented CFMM Typescript API
+
+This package implements a Typescript interface to interact with the Segmented CFMM
+contract.
+
+## Build instructions
+
+You will need the `yarn` package manager to build this library. It can be built
+as follows.
+
+```sh
+yarn install
+yarn tsc
+```
+
+### Generating Typescript interface types
+
+The Typescript types representing the Segmented CFMM contract parameter can be generated
+automatically by using the `make typescript` command.
+The files will be saved to the [/src/generated](src/generated) directory.
+
```

Comment:
Probably it would be good to say that `stack` is needed for this.

Reply:
Added description about `stack`.